content/800-data-platform/200-projects/100-create.mdx

```diff
@@ -96,14 +96,95 @@ As a result, [Data Browser](/data-platform/data-browser) and [Query Console](/da
 
 The platform introspects the database to read the data models so that it can generate the Prisma schema. The Prisma schema enables the features in [Data Browser](/data-platform/data-browser) and [Query Console](/data-platform/query-console), where you can edit and query your data.
 
+#### If introspection fails and can't generate a schema
+
+You will be prompted a notification telling that your project was created but introspection failed. This may be because the database is unreachable or run into a timeout. In order to utilize the Data Browser or Query Console, a project must have a valid Prisma schema that matches with the provided database.
+
+Upon introspection failure, you will be taken to a "Provide schema" view, where you can [provide a schema](#optional-provide-a-schema) by either linking a schema from a repository or upload your schema.
+
+As a result, [Data Browser](/data-platform/data-browser) and [Query Console](/data-platform/query-console) are disabled. You can start using those after you provide a schema. You can [provide a schema](#optional-provide-a-schema) by either linking a schema from a repository or uploading your schema.
```

Comment:
Maybe we could remove this paragraph completely and add the Query Console and Data browser links to the paragraph above. WDYT? 

This paragraph kinda repeats topics mentioned above.